index.js

```diff
@@ -53,11 +58,13 @@ module.exports = {
     app.use(config.nodered.httpAdminRoot, RED.httpAdmin);
     app.use(config.nodered.httpNodeRoot, RED.httpNode);
 
-    if (config.nodered.httpServer)
+    if (config.nodered.httpServer) {
```

Comment:
if, inside if blocks we have only single command - we don't use brackets! (please check linter rules)